packages/xo-server/src/xapi/mixins/networking.js

```diff
@@ -25,6 +25,20 @@ export default {
   async disconnectVif(vifId) {
     await this._disconnectVif(this.getObject(vifId))
   },
+  async editNetwork(
+    id,
+    { automatic, defaultIsLocked, nameDescription, nameLabel, otherConfig }
+  ) {
+    await this._setObjectProperties(this.getObject(id), {
+      defaultIsLocked,
+      nameDescription,
+      nameLabel,
+      otherConfig: {
+        ...otherConfig,
+        automatic: String(automatic),
+      },
```

Comment:
Only use `_setObjectProperties` to update scalar properties.

Use `_updateObjectMapProperty` to update map properties.